test-fresh/src/components/RadialNode.js

```
@@ -0,0 +1,144 @@
+    return null;
+  }
+
+  if (!is.number(x) || !is.number(y)) {
+    if (__DEV__) {
+      console.warn('RadialNode: x and y props must be numbers');
+    }
+    return null;
+  }
+
+  // Floating animation values
+  const floatY = useSharedValue(0);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)